app/views/posts/index.html.erb

```diff
@@ -1 +1,38 @@
-<h1> Check this blogifyers posts</h1>
\ No newline at end of file
+<main>
+  <div class="card">
+    <h1><%= @user.name %></h1>
+    <h3>Number of posts: <%= @user.posts_counter %></h3>
+  </div>
+  <div>
+    <ul>
+      <% @posts.each do |post| %>
+        <%= link_to user_post_path(user_id: post.author_id, id: post.id) do %>
+        <div class="postContainer">
+          <li class="postCont">
+            <div class="post"> 
+              <h2 class="postTitle"><%= post.title %></h2>
+              <h4 class="postText"><%= post.text %></h4>
+              <div class="postCounters">
+                <p class="countText"> Comments: <%= post.comments_counter %> - Likes: <%= post.likes_counter %></p>
+              </div>
+            </div>
+          </li>
+          <div class="postCont">
+              <h3>Comments:</h3>
+              <ul>
+                <li>
+                  <% recent_comments = post.comments.last %>
+                  <% if recent_comments %>
+                    <p class="commentText"> <%= recent_comments.author.name %> : <%= recent_comments.text %> </p>
+                  <% else %>
+                    <p class="commentText"> No comments yet. </p>
+                  <% end %>
```

Comment:
- [ ] Kindly note you need to render the 5 most recent comments of the post, here  I can see only one last comment of the post. Kindly fix this by using the method `recent_comments` in your model

![image](https://github.com/federicaulzurrun/Blogify/assets/79058364/b8c3f988-e04f-49da-a96c-80938d3926ea)
![image](https://github.com/federicaulzurrun/Blogify/assets/79058364/25574aa0-323f-4856-9b72-af2bdb4fb685)